src/Storages/StorageS3.cpp

```diff
@@ -42,6 +52,11 @@
 
 namespace fs = std::filesystem;
 
+#include <boost/algorithm/string.hpp>
+
+
+static const String PARTITION_ID_WILDCARD = "{_partition_id}";
```

Comment:
If string already "escaped" with `{}`, do we nedd `_` at the beginning. Looks like overkill, maybe just `{partition_id}`. But actually I'm not sure which is better.

Reply:
I did that to match existing virtual column name, additionally saving
future possibility to implement expressions or formats (may be).